spec/models/rook_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.describe Rook, type: :model do
+  describe "#valid_move?" do
+    let(:rook) { Rook.create(row: 0, col: 1) }
+
+    it "should allow legal vertical moves" do
```

Comment:
It would be more meaningful if you checked a move several squares away. Otherwise this test is too similar to the King spec.